alerts-server/app/com/alexitc/coinalerts/data/anorm/interpreters/FixedPriceAlertFilterSQLInterpreter.scala

```diff
@@ -40,24 +40,28 @@ class FixedPriceAlertFilterSQLInterpreter {
   }
 }
 
-/**
- * [[sql]] should be a valid SQL executable by anorm, valid SQL could be:
- * - empty string (no conditions required)
- * - string starting with "WHERE" like "WHERE triggered_on IS NULL"
- *
- * The [[sql]] could require unsafe arguments which are provided in the
- * [[params]] list.
- */
-class WhereCondition private[interpreters] (val sql: String, val params: List[(String, String)])
-object WhereCondition {
-  val Empty = new WhereCondition("", List.empty)
-}
+object FixedPriceAlertFilterSQLInterpreter {
+
+  /**
+    * [[sql]] should be a valid SQL executable by anorm, valid SQL could be:
+    * - empty string (no conditions required)
+    * - string starting with "WHERE" like "WHERE triggered_on IS NULL"
+    *
+    * The [[sql]] could require unsafe arguments which are provided in the
+    * [[params]] list.
+    */
+  class WhereCondition private[interpreters](val sql: String, val params: List[(String, String)])
```

Comment:
I'm not convinced on this change, while I get your point, the `WhereCondition` doesn't necessarily belongs to the `FixedPriceAlertFilterSQLInterpreter` but is more like an AST-like.